docs/guides/frontity-query-options.md

```diff
@@ -38,3 +38,7 @@ However packages can also make use of them, for example:
 Currently these are the only Frontity Query Options that Frontity itself uses but it's intended that more will be added in the future. However, you can also add your own Frontity Query Options if you want to use them in your package or theme.
 {% endhint %}
 
+{% hint style="info" %}
```

Comment:
I would change the "hint" style to `warning` or `danger` in this case
https://docs.gitbook.com/editing-content/markdown#hints-and-callouts

Reply:
Good call! Done. 👍